tests/_core/test_data.py

```diff
@@ -187,8 +187,9 @@ def test_index_alignment_between_series(self):
 
         p = PlotData(None, {"x": x, "y": y})
 
-        x_col_expected = pd.Series([10, 20, 30, np.nan, np.nan], np.arange(1, 6))
-        y_col_expected = pd.Series([np.nan, np.nan, 300, 400, 500], np.arange(1, 6))
+        idx_expected = np.arange(1, 6, dtype=pd.RangeIndex(0).dtype)
```

Comment:
Would it be less roundabout to use `np.intp` here? It is meant to represent the platform-appropriate indexing integer.

Reply:
No strong preference just thought it looked weird to instantiate an index just to grab its dtype. You could also just use a `RangeIndex` as the `idx_expected`...

Reply:
I eventually went for the simplest variant `[1, 2, 3, 4, 5]` which corresponds to the indices used for the series.